s/elements/bindings-editor/views/easy-editor-panel.ts

```diff
@@ -0,0 +1,51 @@
+
+import {html} from "lit"
+import {view} from "@chasemoskal/magical/x/view/view.js"
+
+import {Bindings} from "../../../types.js"
+import {hackGetter} from "./easy/hack-getter.js"
+import {buttonLabels} from "../utils/constants.js"
+import {AssignKeybind, Waiting} from "../types.js"
+import {renderKeybind} from "./easy/render-keybind.js"
+import {setupListenToInputsAndActuateKeyBindAssignment} from "../utils/setup-listen-to-inputs-and-actuate-key-bind-assignment.js"
+
+export const EasyEditorPanelView = view(use => ({
+		bindings,
+		eventTarget,
+		onResetDefaults,
+		onKeybindAssignment,
+	}: {
+		bindings: Bindings
+		eventTarget: EventTarget
+		onResetDefaults: () => void
+		onKeybindAssignment: AssignKeybind
+	}) => {
+
+	const [waiting, setWaiting]
+		= use.state<undefined | Waiting>(undefined)
+
+	const getWaiting = hackGetter(use, waiting)
```

Comment:
this `hackGetter` thing is an unsightly blemish, we'll have to deal with it until chase-moskal/magical#2

Reply:
```ts
  //                         (1)
  //                          v
const [waiting, setWaiting, getWaiting]
  = use.state<undefined | Waiting>(undefined)
```

1. once the magical fix is in, we'd do this instead of the hack